### ISSUE TITLE
Accessibility issue: Elements must meet minimum color contrast ratio thresholds on /notebook-ci-testing/notebooks/HASP/Setup/Setup.html

### ISSUE THREAD

**Issue by dulude** [spacetelescope/notebook-ci-testing#177]
## What
An accessibility scan flagged 5 elements on https://spacetelescope.github.io/notebook-ci-testing/notebooks/HASP/Setup/Setup.html because elements must meet minimum color contrast ratio thresholds. Learn more about why this was flagged by visiting https://dequeuniversity.com/rules/axe/4.11/color-contrast?application=playwright.

The following elements need attention:

- `<span class=" -Color -Color-Bold -Color-Bold-Yellow">CI detected...</span>` (selector: `.-Color-Bold-Yellow`)
- `<span class=" -Color -Color-Bold -Color-Bold-Green">2 channel Terms of Service accepted</span>` (selector: `.-Color-Bold-Green`)
- `<span class=" -Color -Color-Red">ERROR: Invalid requirement: &apos;./hasp/.&apos;: Expected package name at the start of dependency specifier</span>` (selector: `.-Color-Red.-Color:nth-child(2)`)
- `<span class=" -Color -Color-Red">    ./hasp/.</span>` (selector: `.-Color-Red.-Color:nth-child(3)`)
- `<span class=" -Color -Color-Red">Hint: It looks like a path. File &apos;./hasp/.&apos; does not exist.</span>` (selector: `.-Color-Red.-Color:nth-child(5)`)

To fix this, ensure the contrast between foreground and background colors meets wcag 2 aa minimum contrast ratio thresholds.

Specifically:

Fix any of the following:
-   Element has insufficient color contrast of 1.76 (foreground color: #ddb62b, background color: #f3f4f5, font size: 10.5pt (14px), font weight: bold). Expected contrast ratio of 4.5:1

## Acceptance Criteria
- [ ] The specific violation reported in this issue is no longer reproducible.
- [ ] The fix MUST meet WCAG 2.2 guidelines OR the accessibility standards specified by the repository or organization.
- [ ] A test SHOULD be added to ensure this specific violation does not regress.
- [ ] This PR MUST NOT introduce any new accessibility issues or regressions.
